pom.xml

```diff
@@ -157,6 +182,26 @@
             <version>1.6.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
`com.google.guava:guava` is definded in parent pom.xml. This declaration may be removed.